.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        php: [7.4, 8.0]
+        php: [7.4, 8.0, 8.1]
```

Comment:
They should allow for a more abstract syntax like  `[7.4, */10, 8.1]` (ie, [min, increment, max]. But I guess that could lead to enormous matrices pretty quickly.